src/Readers/ODataReader.v4/OdcmReader.cs

```diff
@@ -406,6 +411,11 @@ private void WriteProperty(OdcmClass odcmClass, IEdmEntitySet entitySet)
                 {
                     Class = odcmClass,
                     Type = ResolveType(entitySet.EntityType().Name, entitySet.EntityType().Namespace),
+                    Projection = new OdcmProjection()

```

Comment:
Shouldn't we be de-duping Projections here? I think we'd be safe to assume that so long as two Properties require the same capabilities then it's safe to re-use an existing projection.